src/Sweep.cpp

```diff
@@ -291,131 +291,246 @@ void InfectSweep(double t, int run) //added run number as argument in order to r
 	double fp; //// false positive
 	unsigned short int ts;
 
-	if (!P.DoSeasonality)	seasonality = 1.0;
-	else					seasonality = P.Seasonality[((int)t) % DAYS_PER_YEAR];
+	// if not doing seasonality
+	if (!P.DoSeasonality)
+	{	
+		// set seasonality to 1
```

Comment:
I don't think this sort of comment (where the line of code is replicated in english) adds anything useful.

Reply:
In an ideal situation I'd agree with you. My reasoning for doing so was that the variable names don't always reflect what they represent so I felt there was value in spelling out explicitly what was occurring. I'm not wedded to the idea all that strongly though.